### PR TITLE
[ja]: Update translate

### PR DESCRIPTION
**Pull request by zigzagdev**
Sorry to keep pestering you.

maybe meaning of translation is a little wrong.
So, i again to make PR .

I hope this fix helps.

### CODE REVIEW

File: locales/ja/json.json [hunk 1]
```diff
@@ -387,7 +387,8 @@
     "Log In": "ログイン",
     "Log Out": "ログアウト",
     "Log out": "ログアウト",
-    "Log Out Other Browser Sessions": "全ての端末からログアウト",
+    "Log Out Other Browser Sessions": "全ての端末をログアウト",
+    "Log Out of Other Browser Sessions": "全ての端末からログアウト",
```
Comment (andrey-helldar) [inline locales/ja/json.json:391]: No new keys need to be added.

**Review by andrey-helldar — CHANGES_REQUESTED** [review]

File: locales/ja/json.json (hunk 1 above)
Comment (zigzagdev) [reply locales/ja/json.json]: @andrey-helldar 

Hi!

Thanks for reviewing my PR.
Sorry for making new keys.....
deleted it, so please review again !

Comment (andrey-helldar) [reply locales/ja/json.json]: In this project we do not create keys ourselves. The list of keys for translations fully corresponds to the available translations of all official Laravel products.

**Review by andrey-helldar — APPROVED** [review]

File: locales/ja/json.json (hunk 1 above)
Comment (zigzagdev) [reply locales/ja/json.json]: @andrey-helldar 

Ok. I got it.
Thanks for replying for my proposal ! 🙌

**Review by wadakatu — COMMENTED** [review]
@zigzagdev 
PR作成ありがとうございます！

今回の変更点は明確な間違いではなさそうなので変更なしとさせてください🙇
(どちらの表現も散見される)

### 参考資料

https://support.google.com/mail/answer/8154?hl=ja&co=GENIE.Platform%3DDesktop&oco=1&sjid=80461899088022393-AP

https://help.disneyplus.com/ja/article/disneyplus-log-out-devices

**Pull request closed without merging**